core/gallery/gallery.go

```diff
@@ -70,6 +70,20 @@ func DownloadManifest(url string) (info *pb.RpcDownloadManifestResponseManifestI
 	return info, nil
 }
 
+func IsInWhitelist(url string) bool {
+	if len(whitelist) == 0 {
+		return true
+	}
+	// nolint:errcheck
+	parsedURL, _ := uri.ParseURI(url)
```

Comment:
don't ignore error here, if url is parsed with error then it definitely invalid

Reply:
Fixed